### DISCUSSION TITLE
Only show playlist scores on leaderboard after watching replay in playlist

### DISCUSSION THREAD

**Discussion by sz-da** [ppy/osu#26614] [Ideas]
Before watching a replay, the results screen will display the playlist leaderboard.
<img src="https://github.com/ppy/osu/assets/76718358/078b5543-e86f-444a-8626-43df0b0722ae" width="60%">

But after watching a replay, the results screen will display the map's global leaderboard.
<img src="https://github.com/ppy/osu/assets/76718358/f1a006fb-10bd-4a4c-b9e7-b4c079795690" width="60%">

imo it would make more sense to also show the playlist leaderboard after watching a replay

edit: added screenshots

**Comment by mcendu** [comment]
In addition, watching a replay will exit from the playlist to the main menu.

**Reply by sz-da** [reply]
being tracked here https://github.com/ppy/osu/issues/26666
